apps-devstg/us-east-1/databases-mysql --/rds-export-to-s3/main.tf

```diff
@@ -0,0 +1,134 @@
+locals {
+  bucket_name = "${var.project}-${var.environment}-rds-exported-snapshots"
+  tags = {
+    Name        = "rds-export-to-s3"
+    Terraform   = "true"
+    Environment = var.environment
+  }
+}
+
+# -----------------------------------------------------------------------------
+# RDS Export To S3
+# -----------------------------------------------------------------------------
+module "rds_export_to_s3" {
+  source = "github.com/binbashar/terraform-aws-rds-export-to-s3.git?ref=v0.4.0"
+
+  # Set a prefix for naming resources
+  prefix = "mysql"
+
+  create_customer_kms_key = true
+
+  # The database name whose RDS snapshots will be exported to S3
+  database_names = data.terraform_remote_state.databases-mysql.outputs.bb_reference_db_id
+
+  # The RDS snapshots events that should be included: RDS Aurora (RDS-EVENT-0169) or RDS non-Aurora (RDS-EVENT-0091)
+  rds_event_ids = "RDS-EVENT-0169"
```

Comment:
I believe this should pass the event IDs of non-Aurora which should be `RDS-EVENT-0091` (automated snapshots) and `RDS-EVENT-0042` (manual snapshots, if they are supported as well).